packages/components/src/components/hds/badge/index.ts

```diff
@@ -144,3 +148,11 @@ export default class HdsBadgeIndexComponent extends Component {
     return classes.join(' ');
   }
 }
+
+declare module '@glint/environment-ember-loose/registry' {
```

Comment:
In other components this is not added to the file, which one of the two approaches is "correct"?

https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/card/container.ts

Reply:
ah my apologies. I definitely missed this when going over the work that was already in here. You're correct that it can be removed.